### PR TITLE
Added Guidelines.md 

### LINKED ISSUE #105: Guidelines.md for Hacktoberfest 💯
Guidelines for upcoming hacktoberfest for the contributers that would be associated with our project 🥇

### PR DESCRIPTION
**Pull request by unnati914**
Signed-off-by: unnati914 <69121168+unnati914@users.noreply.github.com>
Fixes #105

### CODE REVIEW

File: .github/hacktoberfest 2021/Guidelines 2021.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Guidelines for Hactoberfest 2021 ⚡
```
Comment (joewxboy) [inline .github/hacktoberfest 2021/Guidelines 2021.md:1]: Can/should we include the Open Horizon logo or an icon representation of it here?

BTW, project logo and all artwork lives under https://github.com/open-horizon/artwork

File: .github/hacktoberfest 2021/Guidelines 2021.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Open Horizon 
+Open Horizon is a platform for managing the service software lifecycle of containerized workloads and related machine learning assets. It enables autonomous management of applications deployed to distributed webscale fleets of edge computing nodes and devices without requiring on-premise administrators.
+
+## What is Hactoberfest? 🌱
+
+Hosted by DigitalOcean for the 8th year in a row, Hacktoberfest encourages participation in giving back to the open source community by completing pull requests, participating in events, and donating to open source projects.
+
+All backgrounds and skills levels are encouraged to complete the challenge.
+
+## What is there for you in it?🤝
```
Comment (joewxboy) [inline .github/hacktoberfest 2021/Guidelines 2021.md:11]: Seems like spacing missing between ? and icon compared to all other similar H2 lines in the doc.

File: .github/hacktoberfest 2021/Guidelines 2021.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- free swags
```
Comment (joewxboy) [inline .github/hacktoberfest 2021/Guidelines 2021.md:12]: "swags" should probably be singular/plural "swag" unless you're using it intentionally incorrect ironically.

File: .github/hacktoberfest 2021/Guidelines 2021.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Tech Stacks and technologies that open-horizon uses:
+Open-Horizon uses multiple tech stacks and technologies
+Technologies and Tech stacks listed below:
+- GO
+- Scala
+- Jekyll
+- Markdown
+- Make
+- Travis
```
Comment (joewxboy) [inline .github/hacktoberfest 2021/Guidelines 2021.md:33]: We can probably remove Travis since we're attempting to deprecate

**Comment by unnati914** [comment]
sure I will do the changes. And include some things in it too

**Review by joewxboy — APPROVED** [review]

**Pull request merged**